packages/nextjs/components/pg-ens/LargeGrantProgressBar.tsx

```diff
@@ -0,0 +1,39 @@
+export type GrantProgressBarProps = {
+  className?: string;
+  amount: number;
+  withdrawn?: number;
+  available?: number;
+};
+
+export const LargeGrantProgressBar = ({
```

Comment:
Can be ignored, just to think about:

maybe instead of duplicating this component just add `isLargeGrant` prop?

Reply:
Yeah, makes sense, they are almost the same.

Done!